src/cmd-buildprep

```diff
@@ -40,6 +40,10 @@ def main():
     builds = None
     if fetcher.exists('builds.json'):
         fetcher.fetch_json('builds.json')
+        # Record the origin and original state
+        with open(BUILDFILES['sourceurl'], 'w') as f:
+            f.write(args.url + '\n')
```

Comment:
Do we want `os.path.join(args.url, 'builds.json')` instead here?

Reply:
The intent was to track the "base" URL so it'd be easier to compare later.